agent/aws_iid_attestor.go

```diff
@@ -0,0 +1,175 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"sync"
+
+	"github.com/hashicorp/go-plugin"
+	"github.com/hashicorp/hcl"
+
+	"github.com/spiffe/spire/proto/agent/nodeattestor"
+	"github.com/spiffe/spire/proto/common"
+	spi "github.com/spiffe/spire/proto/common/plugin"
```

Comment:
How are these dependencies managed/pulled in? Almost certainly want to pin somehow. SPIRE uses `glide`, but you can check `vendor` dir in directly if you want.